docs/contributing.rst

```diff
@@ -1,4 +1,197 @@
+  linked to the GitHub issue you created.
+* Reply to the issue with the link to an OSF project component, which is where
+  the final analysis files will be shared. You will need to log into OSF and
+  request access to that component, so that you can upload your files later.
+
+
+
+Carry out the analysis
+^^^^^^^^^^^^^^^^^^^^^^
+
+Run your analysis, but please bear in mind reproducibility and ease of use of
+the output files. Essentially, please be sensible, imagine you are the end-user
+of your data and think about the format that you would want it in.
+
+Please:
+
+* Take note of all commands that were run and software versions. This
+  information can be added to the methods on Github later.
+* Track all samples using the SAM* accession.
+* Keep track of which samples/releases you ran on. At the time of writing
+  there is release 0.2 plus incremental release 2024-08.
+* We will require a "status" file that lists each sample and a result
+  of "PASS", "FAIL", "NOT DONE", so that users can easily find out the overall
+  state of any sample with respect to your analysis.
+* Where possible, keep raw output from tools, but concatenate files into one
+  output file instead of sharing millions files. But make sure the samples and
+  their results are identifiable. For example, if the file is csv/tsv,
+  then add a Sample column and fill this with the SAM* accession.
+* Compress final files with gzip or xz (with the -9 option for maximum
+  compression).
+* Where possible and it makes sense to do so, keep the number of files and
+  total size to a minimum.
+
+
+
+Add files to OSF
+^^^^^^^^^^^^^^^^
+
+You should have been given access to an OSF project component,
+where you can upload results files. Please include an overall "status" file (as
+described above) together will all the other results.
+
+OSF has a file size limit of 5GB, and a project component limit of 50GB.
+We can work around the 50GB limit by batching files over more than one
+component. Request this on your GitHub issue if it is needed and we can create
+more components. For example, see the
+`assembly files <https://osf.io/zxfmy/>`_,
+which are split over two components.
+
+If possible, please organise the files inside the component by having
+top-level folders named with the version number of the tool that was run.
+Inside there, have a folder of each batch of data the analysis was run
+on (for example, release
+0.2, incremental release 2024-08), and also a folder that has
+all results aggregated together, so it is easy for the user to get them.
+This of course may depend on your analysis, for example the
+`AMRfinderPlus results <https://osf.io/7nwrx/>`_ have an extra layer
+of versions because there is the tool version, but also the version of
+the reference database that was used.
+
+
+The intention is that reproducibility files are kept
+on GitHub, however some files will be too big. Please note:
+
+* If you used a container that is not publicly available (eg biocontainers or
```

Comment:
maybe add a point for databases too, the AMRFinder DB is small so this is fine but the bakta DB is massive so probably not possible to upload